src/index.js

```diff
@@ -155,8 +155,8 @@ function generateAndSendSchema(opts) {
       framework_version: expressVersion,
       orm_version: configStore.Implementation.getOrmVersion(),
     };
-    const content = new SchemaFileUpdater(SCHEMA_FILENAME, collections, meta, serializerOptions)
-      .perform();
+    const content = new SchemaFileUpdater({ logger })
```

Comment:
[NOT NEEDED NOW JUST TO INFORM YOU]
You can add `SchemaFileUpdater` inside the context (with `addClass` in `services-builder.js`)
And then use `inject` here to get back your instance. Having all your pieces inside the context is a good thing for reusability. 
Context building also prevent from circular dependencies, alow to test independently stuff, etc..

I set this message as NOT_NEEDED because if you follow the needs of injection, you will have to cover all the application in one PR :)

Reply:
`[NOT NEEDED NOW JUST TO INFORM YOU]` I considered this comment here: https://github.com/ForestAdmin/forest-express/pull/528/commits/73190b1178697af485c91d88495fbcc6b829f527